modules/tensor_mechanics/src/materials/ComputeComplianceTensorBase.C

```diff
@@ -0,0 +1,48 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ComputeComplianceTensorBase.h"
+#include "Function.h"
+
+template <bool is_ad>
+InputParameters
+ComputeComplianceTensorBaseTempl<is_ad>::validParams()
+{
+  InputParameters params = Material::validParams();
+  params.addParam<FunctionName>(
+      "elasticity_tensor_prefactor",
+      "");
+  params.addParam<std::string>("base_name",
+                               "Optional parameter that allows the user to define ");
+  return params;
+}
+
+template <bool is_ad>
+ComputeComplianceTensorBaseTempl<is_ad>::ComputeComplianceTensorBaseTempl(
+    const InputParameters & parameters)
+  : DerivativeMaterialInterface<Material>(parameters),
+    GuaranteeProvider(this),
```

Comment:
This is probably a copy/past remnant, but it doesn't look like you are issuing any material property guarantees. The `GuaranteeProvider` system allows you to declare certain guaranteed properties of the values this object computes (e.g. an elasticity tensor that _is always isotropic_).